core/src/main/java/org/apache/druid/java/util/common/io/Closer.java

```diff
@@ -123,6 +123,17 @@ private Closer()
     return closeable;
   }
 
+  /**
+   * Registers a list of  {@code closeable} to be closed when this {@code Closer} is
```

Comment:
If `{@code closeable}` is a reference to the interface `Closeable`, then it should start with a capital letter. If this is a reference to the parameter, then it should be `closeables`.

Reply:
Makes sense.